### ISSUE TITLE
Accessibility - Dataverse - Color Contrast

### ISSUE THREAD

**Issue by tander08** [QualitativeDataRepository/dataverse#87]
## Homepage: https://data.qdr.syr.edu/
- [x] Dataverse Search Box (including "Search this collection...")
- [x] The dates that appear under each result (for example: Aug 16, 2021 under the first result "the experiences of mothers...")

## (Sample Page) - Data for: Qualitative Data Sharing: Participant Understanding, Motivation, and Consent: https://data.qdr.syr.edu/dataset.xhtml?persistentId=doi:10.5064/F6YYA3O3
- [x] "Version 1.0" under the title
- [x] Read Full Description + Button
- [x] Filter Buttons: Filter By, Filter Type, All, Access, All, File Tag, All,
- [x] Group by Folder; Group by Tag
- [x] All text descriptions under each file name: for example - (under the README file): Plain Text - 3.4KB, Published Aug 12, 2021, 4 Downloads, SHA512..., Documentation Tag. 
- [x] "Download Options" button under the blue "Download Data Project" button
- [x] File Access, Download Options, and Download Metadata under the download menu for each individual data
- [x] Under the Metadata Tab: (Under Related Publication) - the doi link (blue highlight)

## (Sample Page) - Data For: Qualitative Data Sharing: Participant Understanding, Motivation, and Consent - Data - VandeVusee-Mueller_CognitiveInternviewConsentForm.pdf: https://data.qdr.syr.edu/file.xhtml?persistentId=doi:10.5064/F6YYA3O3/ID8HVJ&version=1.0

- [x] Learn about Data Citation Standards
- [x] "Please confirm and/or complete the information needed below in order to continue"
- [x] Accept Button 
- [x] Metadata Tab, File Tag: Documentation

## (Sample Page) - Data For: Monster or Hero? Far-right Responses to Anders Behring Breivik and the 22 July Terrorist Attacks (https://data.qdr.syr.edu/dataset.xhtml?persistentId=doi:10.5064/F6MBCJ8M)

- [x] On the bottom page navigator the box with 1 (currently in blue)

**Comment by qqmyers** [comment]
These should be fixed. Whether the color changes look good or not is another question.  When I made changes, I just changed the lightness of the text color.
Note there are some oddities/false hits:

- The 'Accept' button seems to be complaining about the text shadow color rather than the button background - the contrast of white text on dark blue is >7.
- Any element on a graduated background gets pinged even though the contrast was/is now greater than required between the least extreme end of the gradient (e.g. darkest for dark text).
- Any element where Axe thinks there's overlap it flags and fails. I often see this on the facet panel where some labels get flagged and others don't (same color, same background) which I think is due to one of Schema's footer panels that shifts how much it overlaps with the page as you change the page size (there's no visible effect though).

**Comment by tander08** [comment]
I added another issue that DevTools picked up on a second sample data page (which wasn't present on the first). Since its still picking up some of the issues like you mentioned above, I'm unsure if this is just an extension of that issue or not.

**Comment by qqmyers** [comment]
The last one is another ~false positive - the contrast looks OK but Axe complains about the '1' being too short to decide if it is really text.

**Comment by adam3smith** [comment]
This is a silly one, but WAVE picks up that tthe little > behind "QDR Main Collection" has insufficient contrast.
I'm not even sure what that's doing, but could we make this the same gray as QDR Main Collection?

**Comment by qqmyers** [comment]
I 'fixed' this before reading the comment above. Right now the > is #757575 and the text is #424242; so the > is still lighter. I can make them match if you want (I think #757575 is used elsewhere as that is minimum contrast from an #FFFFFF background.
